src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -140,7 +141,19 @@ trait BeamHelper extends LazyLogging {
             )
           ) ++ {
             if (parsedArgs.useCluster)
-              Map("beam.cluster.clusterType" -> parsedArgs.clusterType.get.toString)
+              Map(
+                "beam.cluster.clusterType" -> parsedArgs.clusterType.get.toString,
+                "akka.actor.provider"      -> "akka.cluster.ClusterActorRefProvider",
+                "akka.remote.artery.canonical.hostname" -> parsedArgs.nodeHost.getOrElse(
+                  throw new RuntimeException("`nodeHost` is not set")
+                ),
+                "akka.remote.artery.canonical.port" -> parsedArgs.nodePort
+                  .map { _.toInt }
```

Comment:
Should we do this in the args parser itself and change the type to an int to start? Otherwise (or in conjunction) should we do something more like a flatMap{x=>Try{x.toInt}.toOption} to handle a non-integer?

Reply:
Changed type to the string and it works, no issue, going to fix it